### PR TITLE
Add janikas-encinitas recommendations

### LINKED ISSUE #8: From Encinitas California
# Why Encinitas?

---
- [x] Beaches
- [x] Sun
- [x] Surf
- [x] Mexican Food

### PR DESCRIPTION
**Pull request by mjanikas**
closes #8 

hey @caroso... check out Encinitas :smile: 

cc: @beardofedu @mfrith

### CODE REVIEW

File: janikas-encinitas.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+- [x] Mexican Food
+
+###Places to Eat
+* Rico's Taco Shop
+* Albertos
+* Robertos
+* Juanitas
+* Cardiff Market
+
+###Places to Go
```
Comment (githubteacher) [inline janikas-encinitas.md:22]: Do you 🏄 ?

**Comment by githubteacher** [comment]
Those places look great! Do you recommend any breweries in the area?

File: janikas-encinitas.md (hunk 1 above)
Comment (mjanikas) [reply janikas-encinitas.md:22]: Used to... but I love the freedom of body surfing... just your fins and the waves

Comment (mfrith) [reply janikas-encinitas.md:22]: I rarely get down there so will have to check them out

File: janikas-encinitas.md [hunk 2]
```diff
@@ -0,0 +1,24 @@
+# Why Encinitas?
+---
+
+- [x] Beaches
+- [x] Sun
+- [x] Surf
+- [x] Mexican Food
+
+###Places to Eat
+* Rico's Taco Shop
```
Comment (caroso) [inline janikas-encinitas.md:10]: What about vegan option 😄 ?

Comment (mjanikas) [reply janikas-encinitas.md:10]: @caroso There you go!

**Comment by mjanikas** [comment]
Added Beer and Vegan option... Good to ship?

@caroso @beardofedu

**Comment by caroso** [comment]
Great! 😄

**Pull request merged**